### ISSUE TITLE
Wie siehts aus? Gibts einen Gewinner/Gewinnerin?

### ISSUE THREAD

**Issue by CarstenKochems** [consistec/connect2013#7]
:)

**Comment by dirkl** [comment]
Hallo, wegen Urlaub hat sich das ein bisschen verzögert & wird im Lauf der Woche geschehen!

**Comment by m1LLo** [comment]
Ist da einer ungeduldig?
;-)

**Comment by m1LLo** [comment]
So jetzt werd ich auch langsam ungeduldig ;-)
Gibt´s denn was neues?

Gruß m1LLo

**Comment by dirkl** [comment]
Ja, der Gewinner steht fest! Die "Unglücklichen" sollten auch schon eine Mail haben, der Gewinner wird informiert, sobald der Liefertermin für das iPad fest ist!

Viele Grüße,

Dirk
